typescript/hardhat/index.ts

```diff
@@ -1,20 +1,40 @@
+import { MultiProvider, TestChainNames } from '@abacus-network/sdk';
 import '@nomiclabs/hardhat-waffle';
+import { ethers } from 'ethers';
```

Comment:
it's less verbose to just import what you need from ethers instead of the base ethers wrapper itself and then drill. E.g. `import {providers, Signers}` instead of `import {ethers}`

Reply:
agreed but this is consistent with usage elsewhere